cmd/mesh/manifest-apply.go

```diff
@@ -37,6 +37,8 @@ type manifestApplyArgs struct {
 	readinessTimeout time.Duration
 	// wait is flag that indicates whether to wait resources ready before exiting.
 	wait bool
+	// yes means don't ask for confirmation (asking for confirmation not implemented)
+	yes bool
```

Comment:
the flag name is not clear enough about what it is doing. BTW there is existing dryrun flag we can probably use instead of creating the new one for confirmation?

Reply:
@richardwxn Good point.  I was thinking of the `--yes` in `apt-get install`.  I considered `--force`.  I considered implementing the "Are you sure" but we don't do that anywhere else in Istio and I didn't want to get this wrong so close to 1.3 release.

`<cmd> mesh apply` has a `--dry-run` flag, but the default is false.  I want some kind of feature that keeps me from changing my cluster if I type this command with no arguments.  Do you have a suggestion?

Reply:
I'd suggest removing dry-run entirely from the commands as it defaults to false.  I think @esnible 's thinking on --yes makes sense given this is how most other distro tools operate.

Reply:
`--dry-run` is nice and familiar.  I propose we keep it.  We still need to agree on a mechanism to make the no-arg default not do any unexpected work.

Reply:
There were previously complaints that dry-run by default is not intuitive and I agree with this. I also think we need to keep dry run as an option. 
My understanding is the concern is only about the no-arg version, as kubectl also lacks confirmation when a file is passed to apply. In that case we could have a CLI prompt for that version of the command only.

Reply:
+1 to keep `--dry-run` as it aims to print result in console only before user decides to apply it to cluster or write to file.

Reply:
@ostromart added "Are you sure?" which only appears if the user supplies no targetting argument.

Reply:
sg. i'll let others approve the PR but overall lgtm.